tools/download-languages.js

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const path = require('path')
+const fs = require('fs')
+const request = require('request')
+
+// The names of the directories in the locales folder are used as a list of languages to retrieve
+var languages = fs.readdirSync(path.join(__dirname, '..', 'app', 'locales')).filter(function (language) {
+  return language !== 'en-US'
+}).map(function (language) {
+  return language.replace('-', '_')
+})
+
+// Setup the credentials
+const username = process.env.USERNAME
+const password = process.env.PASSWORD
+if (!(username && password)) {
+  throw new Error('The USERNAME and PASSWORD environment variables must be set to the Transifex credentials')
+}
+
+// URI and resource list
+const TEMPLATE = 'http://www.transifex.com/api/2/project/brave-laptop/resource/RESOURCE_SLUG/translation/LANG_CODE/?file'
+const resources = [
+  'app', 'menu', 'downloads', 'preferences', 'passwords', 'bookmarks'
```

Comment:
Can this be obtained dynamically based on the files in locales/en-us ?
Would make 1 less step of maintenance as we add new files.